### DISCUSSION TITLE
박지연 - 유기동물 - 보호소로 들어오는 동물의 품종 분석

### DISCUSSION THREAD

**Discussion by yeonjp** [dinnercoding/wanted-pre-onboarding-challenge-data#100] [2023년 11월 프리온보딩 데이터 챌린지]
### 도메인 선정 
- 유기동물의 입양 문제에 관심이 많습니다. 이와 관련해서 해외는 어떻게 돌아가는지 궁금해서 선정하였습니다.
  
### 구체적인 문제 정의
- 한국의 동물보호센터로 입소한 동물의 절반 이상이 사망하거나 보호소에 계속 잔류하고 있습니다. 입양 가는 동물의 품종은 어떤 품종인지, 보호소에 잔류하고 있는 동물의 품종 및 종류는 어떤 종이 있는지 해외 데이터를 분석해보고 싶어 유기동물 데이터를 선정하였습니다.

### 데이터 업로드

- Austin Animal Center Shelter Outcomes (출처 : Kaggle)

**Comment by ghost** [comment]
굉장히 좋은 주제네요! 지연님 :)
평소 본인이 관심 가던 주제를 잘 선정해주셨네요. 데이터 분석을 공부할 땐 평소에 궁금했던 분야를 탐색해보면서 푹 빠져보시는 경험도 정말 좋은 것 같아요!

개인적으로는 우리나라에도 관련 데이터가 있을지 궁금하군요. 해외와 한국의 경우를 비교해봐도 좋을 것 같아요!(저의 아이디어라 꼭 그렇게 하셔야 되는 것은 아닙니다. ㅎㅎ)
